src/EditorFeatures/VisualBasic/RenameTracking/RenameTrackingCodeFixProvider.vb

```diff
@@ -1,41 +0,0 @@
-﻿' Licensed to the .NET Foundation under one or more agreements.
-' The .NET Foundation licenses this file to you under the MIT license.
-' See the LICENSE file in the project root for more information.
-
-Imports System.Composition
-Imports Microsoft.CodeAnalysis.CodeFixes
-Imports Microsoft.CodeAnalysis.Editor.Host
-Imports Microsoft.CodeAnalysis.Editor.Implementation.RenameTracking
-Imports Microsoft.VisualStudio.Text.Operations
-
-Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.RenameTracking
-    ' TODO: Remove the ExtensionOrder attributes once a better ordering mechanism is available
-
-    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeFixProviderNames.RenameTracking), [Shared]>
-    <ExtensionOrder(After:=PredefinedCodeFixProviderNames.AddImport)>
-    <ExtensionOrder(Before:=PredefinedCodeFixProviderNames.AddMissingReference)>
```

Comment:
like with C#, we ordered rename after add-using.  We still preserve that after this PR.